### DISCUSSION TITLE
SSL certificates reload, start/stop appender

### DISCUSSION THREAD

**Discussion by ffatghub** [logfellow/logstash-logback-encoder#947] [Q&A]
Hi All, 
I have a custom appender that makes use of SSL/TLS. It builds SSLContext in the start() phase. In order to reload expired certificates I'm planning to call start/stop within the context of append()... is this OK? Should I schedule a runnable and call start/stop in a different context?
thank you

**Comment by philsttr** [comment]
Is this question related to logstash-logback-encoder somehow?  Does your custom appender extend one of the appenders in logstash-logback-encoder?

**Comment by GabrielJuliao** [comment]
pls see: https://stackoverflow.com/questions/75193024/how-to-reload-logback-ssl-appenders-keystore-truststore-without-restarting-the-a/75193057#75193057
